notebooks/kubeflow_pipelines/walkthrough/solutions/kfp_walkthrough_vertex_pytorch.ipynb

```diff
@@ -0,0 +1,1517 @@
+{
```

Comment:
<ul><li>"Here we define the actual preprocessing logic."  --&gt; ""We now define the actual preprocessing logic:" </li></ul><p> </p><ul><li>"Standardization and One-hot encoding have two steps: fitting to the entire dataset and computing states (e.g., mean and standard deviation for standardization and vocabulary lists for one-hot encoding) and transforming actual values." --&gt; "We will normalize the numerical features and create a one-hot-encoded version of the categorical features. Both feature transformations have two steps: 1) a <code>fit</code> step where we process the whole dataset and compute "states" or statistics needed by the transformations (e.g. mean and standard deviation for the numerical features and value-integer mapping for the categorical features); 2) a <code>transform</code> step where the feature transformations are actually applied"  </li><li>"serializa_constans," --&gt; "serialize_constants"</li></ul><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Thank you for the detailed suggestion. I updated following it, and modified the syntax a bit. </p>